app/views/profiles/edit.mobile.haml

```diff
@@ -2,13 +2,14 @@
 -# licensed under the Affero General Public License version 3 or later. See
 -# the COPYRIGHT file.
 
-= render "shared/settings_nav"
+.stream_element
```

Comment:
That class doesn't look correct in this context. Maybe think of a new name and apply the same styles? (Or pick those that are necessary in this case)